generators/fe_defines.c

```diff
@@ -93,9 +93,23 @@ static void to_upper(char *s) {
 }
 
 static void sanitize_name(char *s) {
+  char *valid_before = "$valid$";
+  char *valid_after = "valid";
+  char *valid_found = NULL;
+
+  while ((valid_found = strstr(s, valid_before)) != NULL) {
+    // overwrite $valid$ with valid, leave out '/0'
+    memcpy(valid_found, valid_after, sizeof(valid_after) - 1);
+
+    // move over rest of string
+    char *src = &valid_found[sizeof(valid_before) - 1];
```

Comment:
`sizeof` will return 8 here I think, which is the size of the pointer 
you need to change the definition of `valid_before` and `valid_after` to `char []`